src/v3/test/integration/__snapshots__/authenticator-enroll-data-phone.test.tsx.snap

```diff
@@ -80,59 +80,63 @@ exports[`authenticator-enroll-data-phone should render form 1`] = `
             <div
               class="MuiBox-root emotion-8"
             >
-              <div
-                class="identifier-container MuiBox-root emotion-9"
+              <form
+                class="o-form MuiBox-root emotion-9"
+                data-se="o-form"
+                novalidate=""
```

Comment:
It seems whereever the previous Identifier container was, it is being replaced with a form, is this intentional?

Reply:
Not intentional, looking into it now. This seems to have started happening after I thought I added the fix for rendering identifier containers in terminal states